net/strongswan/files/gencerts.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
```

Comment:
What?

Reply:
It requires bash.  It won't work with ash.